app/script/conversation/ConversationRepository.js

```diff
@@ -3491,6 +3491,20 @@ z.conversation.ConversationRepository = class ConversationRepository {
         if (conversationEntity && messageEntity) {
           const replacedEntity = conversationEntity.add_message(messageEntity, true);
           if (replacedEntity) {
+            const messages = conversationEntity.messages_unordered();
+
+            const updatedMessages = messages.map(message => {
+              const hasEditedQuote =
+                message.quote && message.quote() && message.quote().messageId === replacedEntity.id;
```

Comment:
```suggestion
                message.quote() && message.quote().messageId === replacedEntity.id;
```

Reply:
I believe the check is legit. We should not remove it

Reply:
:+1: